src/Corporation/Actions.ts

```diff
@@ -463,7 +463,14 @@ export function Research(researchingDivision: Division, researchName: CorpResear
   const researchTree = IndustryResearchTrees[researchingDivision.type];
   if (researchTree === undefined) throw new Error(`No research tree for industry '${researchingDivision.type}'`);
   const research = ResearchMap[researchName];
-
+  const child = researchTree.findNode(researchName);
+  //Check to see if the research request has any pre-reqs that need to be researched first.
+  if (child !== null && child.parent !== null && child.parent.researchName !== null) {
+    const parent = child.parent.researchName;
+    if (!researchingDivision.researched.has(parent)) {
+      throw new Error(`You need ${parent} before you can research ${research.name}`);
```

Comment:
It's not "you" or the player who needs the parent research, it's the researchingDivision who needs it.